doc/user_doc/vic_installation/vch_installer_examples.md

```diff
@@ -0,0 +1,193 @@
+# Examples of Deploying a Virtual Container Host #
+
+This topic provides examples of the `vic-machine` options to use when deploying virtual container hosts in different vSphere configurations.
+
+- [Deploy a Virtual Container Host Directly on an ESXi Host with no Resource Pools](#esxi)
+- [Deploy a Virtual Container Host in a vCenter Server Cluster](#cluster)
+- [Deploy a Virtual Container Host to a Cluster and Specify External, Management, and Container Networks](#networks)
+- [Deploy a Virtual Container Host and Specify a Different Datastore for Container VM Files](#datastores)
+- [Deploy a Virtual Container Host on a Standalone Host in vCenter Server](#standalone)
+- [Deploy a Virtual Container Host in a Resource Pool on an ESXi Host](#rp_host)
+- [Deploy a Virtual Container Host in a Resource Pool in a vCenter Server Cluster](#rp_cluster)
+- [Deploy a Virtual Container Host for Use with a Docker Client that Requires CA Certificates](#ca_cert)
+- [Deploy a Virtual Container Host with no TLS Authentication](#no_tls)
+- [Deploy a Virtual Container Host with Customized VM Settings](#customized)
+
+<a name="esxi"></a>
+## Deploy a Virtual Container Host Directly on an ESXi Host with no Resource Pools##
+
+You can install vSphere Integrated Containers directly on an ESXi host that is not managed by a vCenter Server instance. This example provides the miniumum options required to install vSphere Integrated Containers. The installer prompts you for the password for the ESXi host and deploys a virtual container host appliance with the default name `docker-appliance`. When an ESXi host has no resource pools, you must provide the address of  the root resource pool in the `compute-resource` option in the `govc` format, exactly as shown in the example. You specify the datastore in which to store the virtual container host files and container image files in the mandatory `image-store` option.
```

Comment:
@mdubya66, you mean an end-to-end tutorial type thing? i.e. user starts with nothing, and we walk them step-by-step through multiple processes until they end up with a bunch of containers that do things? That's a different type of doc to the install and admin type docs. Basically, install and admin are task-based, i.e. how do I do task X? Each task/topic should be pretty much a self-contained entity, because people do not read user manuals in a linear fashion. They just go to the bits that help them with their current task at hand.

An end-to-end tutorial is definitely something we could envisage, though, but it's not the same as an install doc or an admin doc, which should be generic rather than tied to a specific example.